instance/tests/integration/integration_instance.py

```diff
@@ -263,6 +247,24 @@ def test_activity_csv(self):
         # stdout should contain 3 lines (as opposed to 2) to account for the last newline.
         self.assertEqual(len(out_lines), 3)
 
+    ''' FIXME JV : Need to add an additional test container?
```

Comment:
One option to free up space is to remove the code for ephemeral databases and remove the ephemeral integration test? Otherwise yeah for now we'd need to add another container I think.